### ISSUE TITLE
Path to /objects/pack is prepended twice

### ISSUE THREAD

**Issue by kluvin** [mirage/irmin#1260]
In a repository of mine I am unable to do anything with the CLI without being stopped by:

```
irmin: [ERROR] ./.git/objects/pack/./.git/objects/pack/pack-<hash>.idx does not exists.
```

*Where the /.git/objects/pack/pack-<hash>.idx can be verified to exist.*

The error disappears if I copy the `.git` either by `git clone --bare` or simple using my CTRL+C-V, although I am only able to set and get new objects, the previously existing objects aren't identified by Irmin.

I have not been able to reproduce the issue.

**Comment by dinosaure** [comment]
I did not look yet, I will look next week :+1: - no tests are available on this part (in `ocaml-git` or `irmin`) to check the regression.

**Comment by dinosaure** [comment]
`ocaml-git` with this fix is not released yet...

**Comment by dinosaure** [comment]
Fixed by ocaml/opam-repository#18048
